draft-ietf-add-split-horizon-authority.xml

```diff
@@ -630,6 +632,15 @@ deeper.subdomain.parent.example. IN AAAA 2001:db8::18
         <t>As in <xref target="internal-only"/>, the internal DNS
         server will need a certificate signed by a CA trusted by the
         client.</t>
+        <t>In order to not leak the internal domains to an external resolver, this document 
```

Comment:
This specification never leaks the internal domains to an external resolver, whether or not they are under a publicly-acknowledged child label.  This might still be good advice, but I think we'll need a different reason for it.

Reply:
Yes, we can only cite the reduction in the frequency of updates to the verification record as an advantage.

Reply:
Suggest:

`Although placing internal domains inside a child domain is unnecessary to prevent leakage, such placement reduces ... [continue with rest of paragraph]`